chai_lab/data/features/generators/msa.py

```diff
@@ -213,17 +213,17 @@ class MSADataSourceGenerator(FeatureGenerator):
 
     def __init__(
         self,
-        num_classes: int = 6,  # TODO how this works with chai1?
+        num_classes: int = 6,  # chai1 : 5 classes + mask val
+        mask_value=5,
```

Comment:
5 maps to `MSADataSource.QUERY`; do we mean to use 4 here instead which maps to `MSADataSource.None`? Might also be better to do `msa_dataset_source_to_int[MSADataSource.NONE]` to avoid such issues. (Might have to make `msa_dataset_source_to_int` a `frozendict` for safety).

Reply:
This logic just reproduces setup used for training. 

Apparently it gets padded with None https://github.com/chaidiscovery/chai-lab/blob/main/chai_lab/data/dataset/msas/msa_context.py#L89, so we can use same here

Reply:
changed to use NONE